auth0/resource_auth0_client_test.go

```diff
@@ -140,6 +141,7 @@ resource "auth0_client" "my_client" {
         callback = "http://example.com/callback"
         slo_enabled = true
       }
+	  signing_cert = "fakecertificate"
```

Comment:
Would it be possible to get close to what a real cert looks like for this test? 🙏🏻  eg
```suggestion
	  signing_cert = "-----BEGIN PUBLIC KEY-----\nMIGf...bpP/t3\n+JGNGIRMj1hF1rnb6QIDAQAB\n-----END PUBLIC KEY-----\n"
```